logging.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"time"
+
```

Comment:
is there suppose to be this empty line in the import?

Reply:
Yes, gofmt does all the formatting.  There's supposed to be a separation from imports from the standard library and third party dependencies